time-series-analyzer-template/src/agent.py

```diff
@@ -0,0 +1,206 @@
+import logging
+import sys
+import threading
+from datetime import datetime, timedelta
+
+import forta_agent
+import pandas as pd
+from forta_agent import FindingSeverity, FindingType, get_json_rpc_url
+from prophet import Prophet
+from web3 import Web3
+
+from src.constants import (ALERT_NAME, BOT_ID, BUCKET_WINDOW_IN_MINUTES,
+                           CONTRACT_ADDRESS, INTERVAL_WIDTH,
+                           TIMESTAMP_QUEUE_SIZE,
+                           TRAINING_WINDOW_IN_BUCKET_SIZE)
+from src.findings import TimeSeriesAnalyzerFinding
+from src.forta_explorer import FortaExplorer
+
+web3 = Web3(Web3.HTTPProvider(get_json_rpc_url()))
+forta_explorer = FortaExplorer()
+
+FINDINGS_CACHE = []
```

Comment:
`ALERTED_TIMESTAMP` isn't initialized globally. 
```suggestion
ALERTED_TIMESTAMP = []
FINDINGS_CACHE = []
```